modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -2974,6 +2977,18 @@ type ObsAttachment {
   fileSize:       Long!
   updatedAt:      Timestamp!
   program:        Program!
+  
+  # All observations to which the attachment is assigned.
+  observations(
+    # Set to true to include deleted values
+    includeDeleted: Boolean! = false
+
+    # Starts the result set at (or after if not existent) the given observation id.
+    OFFSET: ObservationId
+
+    # Limits the result to at most this number of matches (but never more than 1000).
+    LIMIT: NonNegInt
+  ): ObservationSelectResult!
```

Comment:
I took this directly from `type Program`, and copied the elaborator from `ProgramMapping` to `ObsAttachmentMapping`. However, there is a problem here that isn't present in `Program`. This works fine:
`observations(includeDeleted: false)`
but if `includeDeleted` is `true`, `OFFSET` also needs to be included or there is an `AssertionError`. 
I can't figure out what is different in the code. ☹️ 

There is a test where one line can be uncommented to produce the AssertionError. I'll make a comment there.

Reply:
I wonder if making `includeDeleted` optional with a default makes any difference.  (I realize that's not what is done in `Program`, but it seems like it should be.)